clients/unshaded/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -96,7 +96,7 @@ public static synchronized TachyonConf getConf() {
   }
 
   /**
-   * @return the ClientMetrics for this client
+   * @return the {@link ClientMetrics} for this client
```

Comment:
@yupeng9 Currently I use the relative path if the appropriate import is already present. Otherwise I use the full path to avoid an additional import statement. To make it consistent we can switch to the full path for all links. What are your thoughts about this?